pkg/cvo/sync_worker.go

```diff
@@ -585,12 +585,14 @@ func (w *SyncWorker) apply(ctx context.Context, payloadUpdate *payload.Update, w
 	}
 	graph := payload.NewTaskGraph(tasks)
 	graph.Split(payload.SplitOnJobs)
+	var precreateObjects bool
 	switch work.State {
```

Comment:
for those of us less familiar, which other cases are there?  I don't see a default where the precreate would be false.

Reply:
reconciling mode